### DISCUSSION TITLE
Version 1.94.4 updated

### DISCUSSION THREAD

**Discussion by cyring** [cyring/CoreFreq#393] [Show and tell]
CoreFreq version [1.94.1](https://github.com/cyring/CoreFreq/releases/tag/1.94.1) is available for general use and packaging purpose

**Comment by cyring** [comment]
_CoreFreq_ is updated to version `1.94.2`

- [AMD][Raphael] 7950X3D, 7900X3D, 7800X3D, 7900, 7700, 7600
- [AMD][Vermeer] Unlock Ryzen 7 5800X3D
- [UI] Comments added in [`Technologies`] window
- [UI] `HWP` or `CPPC` frequency capabilities moved to the new window [`Perf. Capabilities`]
- [AMD][Rembrandt] Refactored quad channels UMC

**Comment by cyring** [comment]
_CoreFreq_ is updated to version [`1.94.3`](https://github.com/cyring/CoreFreq/releases/tag/1.94.3) to fix build with Debian 4.9.0-19

**Comment by cyring** [comment]
Version [`1.94.4`](https://github.com/cyring/CoreFreq/releases/tag/1.94.4) comes with the followings:
* [UI] Performance Capabilities window size based on `HWP|CPPC` availability
* [AMD] Adding specifics for "Barcelo R" and "Rembrandt R"

**Reply by cyring** [reply]
@inglor @ich777 Hello,

Four your information `1.94.4` is the latest master release.

Regards

**Reply by ich777** [reply]
@cyring thank you, already saw that.
My server notified me yesterday about that and compiled CoreFreq `1.94.4` for Kernel `5.19.17` (latest `stable` Unraid version is based on that Kernel).
You probably see newer Kernel version in my Repository for the CoreFreq Unraid plugin but these newer versions are only from the Unraid `test` branch, in theory I can compile them for these too but that wouldn't make much sense because strictly speaking these are internal releases only and are tested by a small group of people (I only compile versions for `next` and `stable` Unraid releases).
If a new Unraid `test` version is released, the latest available version for CoreFreq will be compiled too, of course if the Kernel was updated too.

Just for clarification. :)

**Reply by cyring** [reply]
Thanks for your explanations. 
Up to now I was indeed looking at   [Unraid-corefreq](https://github.com/ich777/Unraid-corefreq) for newest commit.
